File: launchdarkly/python/src/GetExpiringFlagsForUserExample.py
import json
from datetime import date, datetime
from pprint import pprint

from launchdarkly_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"ApiKey": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    try:
        response = api.UserSettingsApi(api_client).get_expiring_flags_for_user(
            project_key=None,
            user_key=None,
            environment_key=None,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling UserSettings#get_expiring_flags_for_user: %s\n" % e)

File: launchdarkly/php/src/GetExpiringFlagsForUserExample.php
<?php

namespace OSEG\LaunchDarklyExamples;

require_once __DIR__ . '/../vendor/autoload.php';

use SplFileObject;
use LaunchDarkly;

$config = LaunchDarkly\Client\Configuration::getDefaultConfiguration();
$config->setApiKey("ApiKey", "YOUR_API_KEY");

try {
    $response = (new LaunchDarkly\Client\Api\UserSettingsApi(config: $config))->getExpiringFlagsForUser(
        project_key: null,
        user_key: null,
        environment_key: null,
    );

    print_r($response);
} catch (LaunchDarkly\Client\ApiException $e) {
    echo "Exception when calling UserSettings#getExpiringFlagsForUser: {$e->getMessage()}";
}
